t/unit/TestTracker/InteractiveExecutor.t

```diff
@@ -0,0 +1,49 @@
+use strict;
+use warnings;
+
+use Test::More;
+use Sub::Override;
+use Try::Tiny;
+
+use_ok('IPC::Run');
+use_ok('TestTracker::InteractiveExecutor');
+
+&test_argument;
+&test_command;
+
+done_testing;
+
+sub test_argument {
```

Comment:
For this kind of thing, we usually use the ```subtest``` feature of Test::More:
```
subtest 'test arguments` => sub {
    plan tests => 3;
    ...
};
```

But that's just a style change - not necessary